docs/source/markdown/podman-build.1.md.in

```diff
@@ -454,7 +455,8 @@ do not include `History` information in their images.
 
 Set the OS of the image to be built, and that of the base image to be pulled,
 if the build uses one, instead of using the current operating system of the
-build host.
+build host. Unless overridden, subsequent lookups of the image in the local
```

Comment:
I feel like "same image" instead of "image" throughout would be a little bit clearer as to what's going on.

Reply:
Thanks, very nice idea!